build/main.go

```diff
@@ -31,10 +31,10 @@ func main() {
 		GetUserPage:    "/getUser",
 	}
 
-	fileUploadRoutes := &rest.FileUploadHandler{
-		Ctx:            *ctx,
-		UploadFilePage: "/upload",
-	}
+	//fileUploadRoutes := &rest.FileUploadHandler{
```

Comment:
Just delete commented out statements, we can always find them in history

Reply:
Yeah, forgot to remove that I guess.